spec/requests/contacts_spec.rb

```diff
@@ -1,25 +1,27 @@
 RSpec.describe "Contacts", type: :request do
   let(:user) { FactoryBot.create(:user) }
+  let(:contact) { FactoryBot.create(:contact, user:) }
```

Comment:
Should we go with this syntax for factory bot, since we are also using it in other tests?

```suggestion
  let(:user) { create(:user) }
  let(:contact) { create(:contact, user:) }
```